beets/library.py

```diff
@@ -1502,13 +1504,20 @@ def tmpl_aunique(self, keys=None, disam=None):
 
         else:
             # No disambiguator distinguished all fields.
-            res = u' {0}'.format(album.id)
+            res = u' {1}{0}{2}'.format(album.id, bracket[0] if bracket else
+                                       u'', bracket[1] if bracket else u'')
```

Comment:
This is fine—I don't have a strong opinion about whether the brackets should appear here, but you're certainly right that it will look more consistent.